extension/src/helpers/stellar.ts

```diff
@@ -128,11 +128,9 @@ export const isMuxedAccount = (publicKey: string) => publicKey.startsWith("M");
 export const isFederationAddress = (address: string) => address.includes("*");
 
 export const isMainnet = (networkDetails: NetworkDetails) => {
-  const { networkPassphrase, networkUrl } = networkDetails;
+  const { networkPassphrase } = networkDetails;
 
-  return (
-    networkPassphrase === Networks.PUBLIC && networkUrl === NETWORK_URLS.PUBLIC
-  );
+  return networkPassphrase === Networks.PUBLIC;
```

Comment:
Making this change because otherwise, if a user adds their own custom Mainnet connection, it won't be considered `isMainnet` because the networkUrl could be different even though their still connected to Mainnet

Reply:
ah interesting yeah I've wondered about this case before, checking the passphrase seems makes sense to me as long there isn't some way to create another network option and re-use the passphrase which I think is not possible without changing core so 👍